src/account.ts

```diff
@@ -190,3 +196,14 @@ export const importPublic = function(publicKey: Buffer): Buffer {
   }
   return publicKey
 }
+
+/**
+ * Helper method which throws if a string is not hex prefixed
+ * @param {string} input string to check hex prefix of
+ */
+const enforceHexString = function(input: string): void {
+  const msg = `This method only supports 0x-prefixed hex strings but input was: ${input}`
+  if (!ethjsUtil.isHexString(input)) {
+    throw new Error(msg)
+  }
+}
```

Comment:
Using `isHexString` instead of `isHexPrefixed` here because the latter throws it's own error if given a non-string input.